### PR TITLE
Fixed some item typos and add new

### PR DESCRIPTION
**Pull request by BrOgBr**
* **Addressed Issue(s)**: 

* **Server Mode**: Renewal

* **Description of Pull Request**:  Fixed some locations, adding new costumes and improved some scripts.

More coming soon. xD

### CODE REVIEW

File: db/re/item_db.txt [hunk 1]
```diff
@@ -9917,7 +9917,7 @@
 18900,Weisswurst,Weisswurst,4,20,,100,,1,,0,0xFFFFFFFF,63,2,1,,60,0,1022,{ bonus bMaxHP,50; },{},{}
 18901,Seppl_Hat,Seppl Hat,4,20,,500,,5,,1,0xFFFFFFFF,63,2,256,,60,0,1023,{ bonus bVit,3; bonus bHPrecovRate,10; },{},{}
 18902,Shovel_Hat,Shovel Hat,4,0,,500,,0,,0,0xFFFFFFFF,63,2,256,,50,0,1024,{ bonus bVit,3; bonus2 bAddMonsterDropItem,553,500; bonus2 bAddItemHealRate,553,700; },{},{}
-18908,Isabella_Red_Ear,Isabella Red Ear,4,10,,300,,8,,1,0xFFFFFFFF,63,2,256,,1,1,1030,{ bonus bStr,5; bonus bMaxHPrate,5; bonus2 bSubDefEle,Ele_Fire,10; .@r = getrefine(); if (.@r >= 9) bonus bAspd,.@r-8; },{},{}
+18908,Isabella_Red_Ear,Isabella Red Ear,4,10,,300,,8,,1,0xFFFFFFFF,63,2,256,,1,1,1030,{ bonus bStr,5; bonus bMaxHPrate,5; bonus2 bSubDefEle,Ele_Fire,10; .@r = getrefine(); if(.@r>=9) bonus bAspd,1+(.@r-8/2); },{},{}
```
Comment (Atemo) [inline db/re/item_db.txt]: Aspd bonus should be like `bonus bAspd,1+(.@r-9)/2;`

File: db/re/item_db.txt [hunk 2]
```diff
@@ -10331,6 +10358,7 @@
 19834,C_Baseball_Cap,Costume Baseball Cap,4,20,,0,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,216,{},{},{}
 19835,C_Lif_Doll_Hat,Costume Lif Doll Hat,4,20,,0,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,297,{},{},{}
 19836,C_L_Magestic_Goat,Costume Majestic Goat,4,20,,0,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,380,{},{},{}
+19837,C_Asara_Fairy,Costume Asara Fairy Hat,4,10,,0,,,,0,0xFFFFFFFF,63,2,1024,,1,0,492,{},{},{}
```
Comment (Atemo) [inline db/re/item_db.txt]: `Position: Upper, Middle` : 3072
All the costumes are sold to 0, can you change yours to 0?

Comment (BrOgBr) [reply db/re/item_db.txt]: @Atemo I didn't understand, why did you mention about the Position?
Which item is wrong?

Comment (Atemo) [reply db/re/item_db.txt]: According to https://www.divine-pride.net/database/item/19837
the position of this item is 3072
```
	2^10    1024 = Costume Top Headgear
	2^11    2048 = Costume Mid Headgear
```
It should be
`19837,C_Asara_Fairy,Costume Asara Fairy Hat,4,10,,0,,,,0,0xFFFFFFFF,63,2,3072,,1,0,492,{},{},{}`

Comment (BrOgBr) [reply db/re/item_db.txt]: Ok, I'll fix.
Thanks Atemo!

**Comment by Atemo** [comment]
Thanks for your contribution!

**Pull request merged**